backend/.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+  "env": {
```

Comment:
put bracket { on new line it should look like 
``` 
"env" :
{
    "es6": true,
    "node": true
},

Reply:
done